src/dmclock/test/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+INCLUDE (CheckIncludeFiles)
```

Comment:
Since this change is below the src/dmclock subdirectory, unless it's uniquely for ceph it should be applied to the dmclock repo (https://github.com/ceph/dmclock) and then brought into ceph via a git subtree pull (see README.git-subtre).